### PR TITLE
improve tutorials & service init for sendgrid

### LINKED ISSUE #12: Update sendgrid tuto to use new command `service init`
https://docs.mesg.com/tutorials/erc20-transfer-notifications/send-emails-with-sendgrid.html#introduction

### PR DESCRIPTION
**Pull request by ilgooz**
resolves https://github.com/mesg-foundation/docs/issues/12

### CODE REVIEW

**Comment by ilgooz** [comment]
manual tests are ok

File: tutorials/erc20-transfer-notifications/listen-for-transfers-of-an-ethereum-erc20-token.md [hunk 1]
```diff
@@ -17,10 +17,14 @@ We will use the library [Web3.js](https://web3js.readthedocs.io/en/1.0/) to inte
...
-You can find a more advanced and maintained version of this service here: [Service Ethereum ERC20](https://github.com/mesg-foundation/service-ethereum-erc20).
+You can find a more advanced and maintained version of this service here: [Service Ethereum ERC20](https://github.com/mesg-foundation/service-ethereum-erc20)
 
 ::: tip
-MESG Core should already be installed on your computer. If it isn't yet, [install it here](../../guide/start-here/installation.md).
+If you haven't installed **MESG Core** yet, you can do so by running the command:
+
+`bash <(curl -fsSL https://mesg.com/install)`
+
+You can also install it manually by following [this guide](./installation.md#manual-installation).
```
Comment (NicolasMahe) [inline tutorials/erc20-transfer-notifications/listen-for-transfers-of-an-ethereum-erc20-token.md]: wrong link

File: tutorials/erc20-transfer-notifications/receive-email-when-there-is-an-erc20-transfer.md [hunk 2]
```diff
@@ -16,7 +16,11 @@ Today we'll learn how to create a JavaScript application that connects the two p
 You can access the final version of the [source code on GitHub](https://github.com/mesg-foundation/docs/tree/master/tutorials/erc20-transfer-notifications/email-notification-on-erc20-transfer).
 
 ::: tip
-MESG Core should already be installed on your computer. If it isn't yet, [install it here](../../guide/start-here/installation.html).
+If you haven't installed **MESG Core** yet, you can do so by running the command:
+
+`bash <(curl -fsSL https://mesg.com/install)`
+
+You can also install it manually by following [this guide](./installation.md#manual-installation).
```
Comment (NicolasMahe) [inline tutorials/erc20-transfer-notifications/receive-email-when-there-is-an-erc20-transfer.md]: wrong link

File: tutorials/erc20-transfer-notifications/send-emails-with-sendgrid.md [hunk 3]
```diff
@@ -14,12 +14,14 @@ We'll start with the example of an email provider where we will send emails thro
...
 
-You can find a more advanced and maintained version of this service here: [Service Email SendGrid](https://github.com/mesg-foundation/service-email-sendgrid).
+You can find a more advanced and maintained version of this service here: [Service Email SendGrid](https://github.com/mesg-foundation/service-email-sendgrid)
 
 ::: tip
 If you haven't installed **MESG Core** yet, you can do so by running the command:
 
 `bash <(curl -fsSL https://mesg.com/install)`
+
+You can also install it manually by following [this guide](./installation.md#manual-installation).
```
Comment (NicolasMahe) [inline tutorials/erc20-transfer-notifications/send-emails-with-sendgrid.md]: wrong link

File: tutorials/erc20-transfer-notifications/send-emails-with-sendgrid.md [hunk 4]
```diff
@@ -29,24 +31,47 @@ MESG services are composed of two different parts:
...
+:::
 
 ## Send emails through MESG tasks
 
 ### Configure your task
 
-Open the `mesg.yml` file and add a new task called `send` responsible for sending an email based on the following inputs:
+Let's add the task we want to serve for MESG Core to the `mesg.yml` file.
+
+First, clean the `mesg.yml` file, keeping only the keys: `name` and `description`. And declare a `sid` for the service. It should look like this:
```
Comment (NicolasMahe) [inline tutorials/erc20-transfer-notifications/send-emails-with-sendgrid.md]:
```suggestion
First, clean the `mesg.yml` file, keeping only the keys: `name`, `sid` and `description`. Change their values to look like this:
```

File: tutorials/erc20-transfer-notifications/listen-for-transfers-of-an-ethereum-erc20-token.md [hunk 5]
```diff
@@ -159,10 +179,11 @@ Now, it's time to transform this node app to a fully-compatible MESG Service.
 
 Let's add the event we want to emit to MESG Core to the `mesg.yml` file.
 
-First, clean the `mesg.yml` file, keeping only the keys: `name` and `description`. It should look like this:
+First, clean the `mesg.yml` file, keeping only the keys: `name` and `description`. And declare a `sid` for the service. It should look like this:
```
Comment (NicolasMahe) [inline tutorials/erc20-transfer-notifications/listen-for-transfers-of-an-ethereum-erc20-token.md]:
```suggestion
First, clean the `mesg.yml` file, keep only the keys: `name` and `description`. And declare a `sid` for the service. It should look like this:
```

Comment (NicolasMahe) [inline tutorials/erc20-transfer-notifications/listen-for-transfers-of-an-ethereum-erc20-token.md]:
```suggestion
First, clean the `mesg.yml` file, keeping only the keys: `name`, `sid` and `description`. Change their values to look like this:
```

**Review by NicolasMahe — APPROVED** [review]

**Pull request merged**